app/models/submission/validations_by_template_name.rb

```diff
@@ -45,13 +62,11 @@ def apply_additional_validations_by_template_name
   #
   # @param column_header [String] The header of the column to validate.
   # @return [void]
-  # rubocop:disable Metrics/MethodLength,Metrics/AbcSize
+  # rubocop:disable Metrics/MethodLength
```

Comment:
I keep running fowl of this rubocop, I wonder if we should increase the allowed length? it's too short